NanoAnalysis/python/modules/jetJERC.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+def getJetCorrected(era, tag, is_mc, overwritePt=True) :
+    from PhysicsTools.NATModules.modules.jetCorr import jetJERC
+
+    if era != 2022:
+        raise ValueError("getJetCorrected: Era", era, "not supported")
+
+    if is_mc :
+        if tag == "pre_EE":
+            folderKey = "2022_Summer22"
+            L1Key = "Summer22_22Sep2023_V2_MC_L1FastJet_AK4PFPuppi"
+            L2Key = "Summer22_22Sep2023_V2_MC_L2Relative_AK4PFPuppi"
+            L3Key = "Summer22_22Sep2023_V2_MC_L3Absolute_AK4PFPuppi"
+            L2L3Key = "Summer22_22Sep2023_V2_MC_L2L3Residual_AK4PFPuppi"
+            smearKey = "JERSmear"
+            JERKey = "Summer22_22Sep2023_JRV1_MC_PtResolution_AK4PFPuppi"
+            JERsfKey = "Summer22_22Sep2023_JRV1_MC_ScaleFactor_AK4PFPuppi"
+        elif tag == "post_EE":
+            folderKey = "2022_Summer22EE"
+            L1Key = "Summer22EE_22Sep2023_V2_MC_L1FastJet_AK4PFPuppi"
+            L2Key = "Summer22EE_22Sep2023_V2_MC_L2Relative_AK4PFPuppi"
+            L3Key = "Summer22EE_22Sep2023_V2_MC_L3Absolute_AK4PFPuppi"
+            L2L3Key = "Summer22EE_22Sep2023_V2_MC_L2L3Residual_AK4PFPuppi"
+            smearKey = "JERSmear"
+            JERKey = "Summer22EE_22Sep2023_JRV1_MC_PtResolution_AK4PFPuppi"
+            JERsfKey = "Summer22EE_22Sep2023_JRV1_MC_ScaleFactor_AK4PFPuppi"
+    ## Data
+    ## JER are not applied to data
+    else :
+        if tag == "pre_EE":
+            folderKey = "2022_Summer22"
+            L1Key = "Summer22_22Sep2023_RunCD_V2_DATA_L1FastJet_AK4PFPuppi"
+            L2Key = "Summer22_22Sep2023_RunCD_V2_DATA_L2Relative_AK4PFPuppi"
+            L3Key = "Summer22_22Sep2023_RunCD_V2_DATA_L3Absolute_AK4PFPuppi"
+            L2L3Key = "Summer22_22Sep2023_RunCD_V2_DATA_L2L3Residual_AK4PFPuppi"
+            smearKey = None
+            JERKey = None
+            JERsfKey = None
+        elif tag == "E":
```

Comment:
And for a similar reason, I would set these to "2022E", etc. and use `in` instead of `==`.
(We need some care on choosing DATA_TAG strings as several modules use it in different ways.
To keep track of that, I listed which strings are actually searched for [here](https://github.com/CJLST/ZZAnalysis/blob/Run3/NanoAnalysis/python/nanoZZ4lAnalysis.py#L24-L28) - maybe you can add these as well? Note that "post_EE" is not listed as that is used as the default, so no module searches for that as of now)

Reply:
Done. I also added a ValueError for data in case the tag is not among the ones we expect